app/javascript/stylesheets/application.scss

```diff
@@ -1,3 +1,4 @@
-@import "tailwindcss/base";
-@import "tailwindcss/components";
-@import "tailwindcss/utilities";
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
+@import '@fortawesome/fontawesome-free';
```

Comment:
@arnabsen1729 this was not required. We use the FA js to replace the `<i>` elements with SVG on runtime.